### PR TITLE
Clarify binaryData handling by kubectl create and describe

### PR DESCRIPTION
**Pull request by schollii**
Addresses the documentation portion of https://github.com/kubernetes/website/issues/25921

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 1]
```diff
@@ -201,6 +201,8 @@ allow.textmode=true
 how.nice.to.look=fairlyNice
 ```
 
+Note that data sources that are not ASCII or UTF-8 will be put in the `binaryData` field of the configmap. Both text and binary data sources can be combined in one configmap, but the `binaryData` field is not displayed (as of kubectl 1.20) by `kubectl describe`. To see the `binaryData` in the configmap, you can use `kubectl get configmap -o json MAP_NAME`.
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:204]: I'd write:
```suggestion
When `kubectl` creates a ConfigMap from inputs that are not ASCII or UTF-8, the tool puts these into the `binaryData` field of the ConfigMap, and not in `data`. Both text and binary data sources can be combined in one ConfigMap.
If you want to view the `binaryData` keys (and their values) in a ConfigMap, you can run `kubectl get configmap -o jsonpath='{.binaryData}' <name>`.
```
How does that sound?

Comment (schollii) [reply content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:204]: Fine by me!

**Review by irvifa — COMMENTED** [review]
/lgtm

**Pull request merged**